buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/LicenseHeadersTask.groovy

```diff
@@ -178,4 +180,7 @@ class LicenseHeadersTask extends AntTask {
             throw new IllegalStateException("License header problems were found! Full details: " + reportFile.absolutePath)
         }
     }
+
+    @Internal
+    abstract ListProperty<FileCollection> getSourceFolders();
```

Comment:
Should we be using these "managed" properties more often?

Reply:
I think it makes sense to move our build towards using these